auto3dseg/algorithm_templates/dints/scripts/search.py

```diff
@@ -167,9 +172,9 @@ def run(config_file: Optional[Union[str, Sequence[str]]] = None, **override):
         )[dist.get_rank()]
     logger.debug(f"val_files:, {len(val_files)}")
 
-    train_cache_rate = float(parser.get_parsed_content("searching#train_cache_rate"))
+    train_cache_rate = float(parser.get_parsed_content("train_cache_rate"))
     validate_cache_rate = float(
-        parser.get_parsed_content("searching#validate_cache_rate"))
+        parser.get_parsed_content("validate_cache_rate"))
```

Comment:
As as above, it seems to me that "searching#" should be added

Reply:
I have fixed this line.